app/services/status_update_service.rb

```diff
@@ -1,7 +1,4 @@
 class StatusUpdateService < ApplicationService
```

Comment:
Sure, the discussion regarding this is in https://docs.google.com/document/d/1vPoBoakZfLFbvQinyOY6pHtak_BAdrL9Ua0hRqD73L0/edit?disco=AAAAGvVBSDg This change is unfortunately a rather big scope jump for this PR so I can understand the surprise/concern.

What we seem to lack are scenarios where we feel it is our duty to provide this retry facility and given it fails 90% of the time is it worth it? At the moment the example we've got is that a users mailbox is full. I'd consider this more a user problem to not get the email rather than an indication we need to hold it for them.

It wouldn't surprise me hugely if we later find out information that indicates we may want to re-include this - but if that occurs at least then it'll be clear why we need it. Whereas now it's a remnant from the early days of this app where we trying to cover bases on what could go wrong.

Reply:
Thanks for the clarification. Thinking about it I'd hope Notify would retry these instead of us having to house this retry logic as even the terminology 'temporary failure' seems to indicate the email _can_ and _should_ be sent in the future but I see from the doc that we're unsure whether this happens. 

I'm still not 100% keen on removing something that seems to improve our service (albeit it's not very efficient) but as you say if we ever need to re-apply this behaviour we can do in the future.